proposals/csharp-10.0/lambda-improvements.md

```diff
@@ -211,11 +264,12 @@ lambda_parameter
   ;
 ```
 
-## Open issues
-- Inferring a delegate type for lambdas and method groups might result in breaking changes in overload resolution: see [issues/4674](https://github.com/dotnet/csharplang/issues/4674) for examples.
-
 ## Design meetings
 
 - https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-03-03.md
-- https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-04-12.md
+- https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-04-12.md#lambda-improvements
+- https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-04-21.md#inferred-types-for-lambdas-and-method-groups
+- https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-05-10.md
+- https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-06-02.md#lambda-return-type-parsing
+- https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-06-21.md#open-questions-for-lambda-return-types
```

Comment:
Thanks for updating the design meetings list. 
nit: I'd be tempted to move all of the to the championed issue (easier to update), but not crucial...